src/vmware/azext_vmware/_params.py

```diff
@@ -70,3 +70,13 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware hcx-enterprise-site') as c:
         c.argument('name', options_list=['--name', '-n'], help='The name of the HCX Enterprise Site.')
+
+    with self.argument_context('vmware datastore') as c:
+        c.argument('name', options_list=['--name', '-n'], help='The name of the datastore.')
+        c.argument('cluster', help='The name of the cluster.')

```

Comment:
does it support id?

Reply:
It does not. I'm curious what you are recommending.

Reply:
cli convention is to support both resource id and resource name at one param, such as storage_account, to provide convinience for users

Reply:
I created https://github.com/Azure/azure-cli-extensions/issues/3020 and will add it to a 2.1 of vmware ext.